src/main/scala/edu/colorado/plv/cuanto/numdomain/AbsCons.scala

```diff
@@ -0,0 +1,8 @@
+package edu.colorado.plv.cuanto.numdomain
+
+/**
+  * Created by lumber on 4/20/17.
+  */
```

Comment:
Ahem, nit but you did agree to it: https://cuplv.slack.com/archives/C3S635ZD4/p1497045800481119